server/routerlicious/packages/services/src/redis.ts

```diff
@@ -44,18 +44,28 @@ export class RedisCache implements ICache {
 	}
 
 	public async get(key: string): Promise<string> {
-		return this.client.get(this.getKey(key));
+		try {
+			return this.client.get(this.getKey(key));
+		} catch (error) {
+			Lumberjack.error(`Error getting ${key.substring(0, 20)} from cache.`, undefined, error);
+			throw {name: (error as Error)?.name, message: (error as Error)?.message};
+		}
 	}
 
 	public async set(key: string, value: string, expireAfterSeconds?: number): Promise<void> {
-		const result = await this.client.set(
-			this.getKey(key),
-			value,
-			"EX",
-			expireAfterSeconds ?? this.expireAfterSeconds,
-		);
-		if (result !== "OK") {
-			throw new Error(result);
+		try {
+			const result = await this.client.set(
+				this.getKey(key),
+				value,
+				"EX",
+				expireAfterSeconds ?? this.expireAfterSeconds,
+			);
+			if (result !== "OK") {
+				throw new Error(result);
+			}
+		} catch (error) {
+			Lumberjack.error(`Error setting ${key.substring(0, 20)} in cache.`, undefined, error);
+			throw {name: (error as Error)?.name, message: (error as Error)?.message};
```

Comment:
I think if you do `catch (error: Error)` or `catch(error: any)`, you don't need to do the `(error as Error)` part.

Smam comment applies to other methods

Reply:
Fixed